packages/image_picker/tizen/src/image_picker_tizen_plugin.cc

```diff
@@ -0,0 +1,310 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "image_picker_tizen_plugin.h"
+
+#include <app_control.h>
+#include <assert.h>
+#include <flutter/method_channel.h>
+#include <flutter/plugin_registrar.h>
+#include <flutter/standard_method_codec.h>
+#include <system_info.h>
+
+#include "image_resize.h"
```

Comment:
You may move this line to line 25 and also move `tizen/inc/image_resize.h` to `tizen/src/image_resize.h`.

Reply:
done.